### ISSUE TITLE
Installing liquorix with curl on tuxedo os. [request]

### ISSUE THREAD

**Issue by gugg305** [damentz/liquorix-package#116]
Have used liquorix with tuxedo os without problems using ppa. Gives better battery and runs cooler.
Adding it to the curl script would be nice!
[INFO ] Distribution is tuxedo
[ERROR] This distribution is not supported at this time

**Comment by damentz** [comment]
The install script detects the base operating system.  Tuxedo should not be changing the base OS from Ubuntu to Tuxedo as that will break more than this install script.

Does this command work? `sudo add-apt-repository ppa:damentz/liquorix`

Also, what's the output of `lsb_release -a`.?

**Comment by damentz** [comment]
Ok, this should be resolved now.  I poked at a few different distributions that are layers on top of the major 3 (Debian, Ubuntu, Arch), and they all use ID_LIKE in /etc/os-release to define which distribution they're based on.

Updated the install script on the website to account for that, next time you follow the instructions, Liquorix should install correctly.

Commits:
1. [scripts/install-liquorix.sh: Try ID_LIKE first in os-release](https://github.com/damentz/liquorix-package/commit/5e37c393d8f2a3a714dafcdb4ef76def9d2c9805)
2. [scripts/install-liquorix.sh: Evaluate Ubuntu before Debian](https://github.com/damentz/liquorix-package/commit/f3c4d1bc29af1cb2f3988e6b4b6fd388fac19f32)
3. [scripts/install-liquorix.sh: Use wildcard patterns to match distro ID](https://github.com/damentz/liquorix-package/commit/afb64b5cd22f1fa98e1797b3452055c8885656fc)
